src/app/views.py

```diff
@@ -1290,13 +1290,40 @@ def archiveRequests(request, license_id=None):
     """ View for archive license requests
     returns archive_requests.html template
     """
+    context_dict = {}
     if request.method == "POST" and request.is_ajax():
-        archive = request.POST.get('archive', False)
-        license_id = request.POST.get('license_id', False)
-        if license_id:
-            LicenseRequest.objects.filter(pk=license_id).update(archive=archive)
+        if request.user.is_authenticated():       
+            user = request.user
+            """ Getting user info for submitting github issue """
+            github_login = user.social_auth.get(provider='github')
+            token = github_login.extra_data["access_token"]
+            username = github_login.extra_data["login"]
+            test = requests.get('https://api.github.com/repos/spdx/tools-python/collaborators/'+username , headers={'Authorization': 'token {}'.format(token) })
```

Comment:
Going through the code, I found that this would be more suitable if moved in utils.py just like we have function for makePullRequest.

Reply:
Done the changes. I created a function to retrieve the response code. Should I add more functionality to it??